watershed_workflow/__init__.py

```diff
@@ -1185,6 +1187,8 @@ def triangulate(hucs,
         List of objects, whose boundary (in the case of
         polygons/waterbodies) or reaches (in the case of River) will
         be present in the edges of the triangulation.
+    hole_points : list(shapely.Point), optional
+        List of points inside the polygons to be left as holes/voids (excluded from mesh)
```

Comment:
also document refinement_polygons.  I'm not sure what that is?

Reply:
I think that is a depreciated option. I just removed that. One can always define a custom refinement function, right and use polygons or lines in the refinement function as needed?

Reply:
Well, the case we used to support is the case where we want to have the river be edges in the mesh.  I'm not sure that is really ever helpful though, so maybe I'm ok with losing that option.